### PR TITLE
Add error logging method

### PR DESCRIPTION
**Pull request by burkel24**

### CODE REVIEW

File: app/site/main/rubiconmd.js [hunk 1]
```diff
@@ -292,6 +292,54 @@ var RubiconMD = function(options) {
...
+    };
+
+    var url = rootApiUrl() + '/client_logger';
+    var data = JSON.stringify({
+      access_token: token.access_token,
+      events: [ payload ]
+    });
+
+    // Pass the full XHR back
+    fetch(url, RequestTypeEnum.POST, data, function(response) {});
```
Comment (iovis9) [inline app/site/main/rubiconmd.js]: The callback is optional

**Comment by iovis9** [comment]
Should we start enforcing a js linter with a config committed to the project root? I use `eslint`.

**Comment by burkel24** [comment]
Re: the linter, I believe we're going to rebuild this thing in TS before too long - I'd prefer to wait and set up all the build tooling then

**Comment by burkel24** [comment]
Re: the linter, I believe we're going to rebuild this thing in TS before too long - I'd prefer to wait and set up all the build tooling then

**Pull request merged**